.github/workflows/main.yml

```diff
@@ -123,7 +123,8 @@ jobs:
       with:
         fetch-depth: 0
     - name: Set up Python ${{ matrix.python }}
-      uses: actions/setup-python@v2
+      # https://github.com/actions/setup-python/issues/171
+      uses: actions/setup-python@v2.1.4
```

Comment:
a workaround for `Error: IS_WINDOWS is not defined` to get builds green ->   https://github.com/actions/setup-python/issues/171